tests/NTest_adc_env.lua

```diff
@@ -1,7 +1,8 @@
 -- Walk the ADC through a stepped triangle wave using the attached voltage
 -- divider and I2C GPIO expander.
 
-local N = require('NTest')("adc-env")
+local N = ...
+N = (N or require "NTest")("adc-env")
```

Comment:
But how can the "adc-env" be set if the NTest comes from outside?
I wouldn't like to spread such strings over several files

Reply:
See above and below.

Reply:
ok got it now